Selenium_Task80/TestProject2/LoginTests.cs

```diff
@@ -17,7 +19,8 @@ public void BeginTest()
             driver = new ChromeDriver();
             driver.Url = "https://www.yandex.com";
             Screenshot screenshot = ((ITakesScreenshot)driver).GetScreenshot();
-            screenshot.SaveAsFile("./Screenshot/screenshot.jpg", ScreenshotImageFormat.Jpeg);
+            var Path = Directory.GetCurrentDirectory().Replace("TestProject2\\bin\\Debug\\net5.0", "") + "Screenshot\\";
```

Comment:
If net version changes test will fail
If run not in Debug mode test will fail
Try to get index of your project's name and substring for example